meta-dts-distro/recipes-dts/dts/dts/dts

```diff
@@ -103,6 +103,7 @@ while : ; do
           echo "Please consider supporting Dasharo by sending the logs next time."
           ;;
       esac
+      sync_clocks
```

Comment:
> I could not make a full list of the tasks though

Why not? When we need time synchronization?

What if we start DTS in FUM? https://docs.dasharo.com/kb/firmware-update-mode/#firmware-update-mode-flowchart Will time also be correctly synchronized?

Reply:
> Why not? When we need time synchronization?

Tried to analyze possible tasks which would possibly need time to be synced and got to conclusion that I am not experienced enough to cover all possible cases.

> What if we start DTS in FUM? https://docs.dasharo.com/kb/firmware-update-mode/#firmware-update-mode-flowchart Will time also be correctly synchronized?

I do not see any reason time would not be synced, cos FUM seems not to change anything connected with System Clock or connection to network.

Reply:
Launching DTS in FUM will cause clock not to be synced indeed, cos it will not launch `dts` script (from `dts-boot` script):
```bash
#!/bin/bash

SBIN_DIR="/usr/sbin"
FUM_EFIVAR="/sys/firmware/efi/vars/FirmwareUpdateMode-d15b327e-ff2d-4fc1-abf6-c12bd08c1359"

export DTS_FUNCS="$SBIN_DIR/dts-functions.sh"
export DTS_ENV="$SBIN_DIR/dts-environment.sh"

if [ -d $FUM_EFIVAR ]; then
    $SBIN_DIR/dasharo-deploy update fum
else
    $SBIN_DIR/dts
fi

```

Reply:
> Tried to analyze possible tasks which would possibly need time to be synced and got to conclusion that I am not experienced enough to cover all possible cases.

So far it's been problematic with firmware verification only. So why not put it in the update function?

Reply:
Resolving this thread due to moving this PR to [`Dasharo/dts-scripts`](https://github.com/Dasharo/dts-scripts) repo duie to changes in https://github.com/Dasharo/dasharo-issues/issues/773.